webrtc/videoroom/lib/videoroom/display_manager.ex

```diff
@@ -138,7 +138,8 @@ defmodule VideoRoom.DisplayManager do
         {:ok, state}
 
       true ->
-        raise("No such endpoint id #{inspect(endpoint_id)}")
+        Membrane.Logger.debug("No such endpoint id #{inspect(endpoint_id)}")
+        {:ok, state}
```

Comment:
Maybe it would be good idea to change raise to debug log in update/3 function as well. Is that reasonable to crash whole room because we can't update status of peer that actually does not exist?

Reply:
This was the first version but in theory it can't happen so we decided to raise. I would leave it for now

Reply:
ok